src/enclave/enclave_init.c

```diff
@@ -148,6 +148,9 @@ static int startmain(void* args)
         sgxlkl_info("wg0 has public key %s\n", key);
     }
 
+    /* Set locale for usersapce components using it */
+    pthread_t self = __pthread_self();
+    self->locale = &libc.global_locale;
```

Comment:
This setup should be done in the start libc routine in musl.

Reply:
Oh okay, makes sense. I'll look for the appropriate place within musl to set this.